tests/Css/StyleTest.php

```diff
@@ -43,15 +43,36 @@ public static function lengthInPtProvider(): array
             ["calc(50pt*2)", null, 100.0],                 // * do not require whitespace
             ["calc(50%/2)", 120, 30.0],                    // / do not require whitespace
             ["calc(10pt + -50%)", 12, 4.0],                // negative value
-            ["CalC(10)", null, 10.0],                      // case-insensitive
```

Comment:
Why remove that?

Reply:
Mistake, I corrected case-insensitive handling